lib/decoder.js

```diff
@@ -213,6 +224,10 @@ Decoder.prototype.decode = function(len, data, callback) {
     }
   }
 
+  if(len === 11) {
```

Comment:
I would not add assumptions for other DPT types, if there is no unique identification.
With my changes (you just merged), you get the raw data with the UNKN Data Type.

In my projects, I did the following:
I am using a unique naming convention for the group addresses in ETS. For example I use "ea301/1 Light Kitchen" for on / off (ein/aus) in the room number 301, Light 1, Room Kitchen.
Or "tmp301 Kitchen" for the room temperature in the room 301 (Kitchen).

I export the group addresses in a csv.
In my node projects I read the group address csv.
When receiving a read or write request, I do a lookup in my group address list and use the datatype that is identified by the name prefix.